File: Anul 2/Semestrul 2/MPP/labs/Tema 4-proiect Server-Client Csharp/proiectSCcsharp/angajatModel/SpectacolDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace proiectSCcsharp.domain
{
    [Serializable]
    public class SpectacolDTO
    {
        public string artistName { get; set; } 
        public string time { get; set; }
        public string place { get; set; }
        public int availableSeats { get; set; }

        public SpectacolDTO(string ArtistName, string Time, string Place, int AvailableSeats)
        {
            this.artistName = ArtistName;
            this.time = Time;
            this.place = Place;
            this.availableSeats = AvailableSeats;
        }
    }
}

File: Anul 2/Semestrul 2/MPP/labs/Tema 7-servicii REST/Java/Rest/proiect/Domain/src/main/java/proiect/domain/SpectacolDTO.java
package proiect.domain;

import java.io.Serializable;

public class SpectacolDTO {
    private String artistName;
    private String time;
    private String place;
    private int availableSeats;

    public SpectacolDTO(Spectacol spectacol) {
        //super(id);
        this.artistName = spectacol.getArtistName();
        this.time=spectacol.getTime();
        this.place = spectacol.getPlace();
        this.availableSeats = spectacol.getAvailableSeats();
    }
    public String getArtistName() {
        return artistName;
    }

    public void setArtistName(String artistName) {
        this.artistName = artistName;
    }

    public String getTime() {
        return time;
    }

    public void setTime(String time) {
        this.time = time;
    }

    public String getPlace() {
        return place;
    }

    public void setPlace(String place) {
        this.place = place;
    }

    public int getAvailableSeats() {
        return availableSeats;
    }

    public void setAvailableSeats(int availableSeats) {
        this.availableSeats = availableSeats;
    }
}
